app/models/property.rb

```diff
@@ -25,15 +25,39 @@ class Property < ApplicationRecord
 
   validates :city,
             inclusion: { in: Canonical::Property::VALID_CITIES, message: 'must be a Skyrim city in which an ownable property is located', allow_blank: true },
-            uniqueness: { scope: :game_id, message: 'must be unique per game if present', allow_blank: true }
+            uniqueness: { scope: :game_id, message: 'must be unique per game if present', allow_nil: true }
+
+  validate :ensure_alchemy_lab_available, if: -> { has_alchemy_lab == true && canonical_property&.alchemy_lab_available == false }
```

Comment:
We don't want this validation to fail if the `canonical_property` is `nil`, hence why we can't use `has_alchemy_lab == true && !canonical_property&.alchemy_lab_available`.